packages/cyberstorm/src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+import React, { CSSProperties, ReactNode } from "react";
+import styles from "./Toast.module.css";
+import { Icon } from "../Icon/Icon";
+import { classnames } from "../../utils/utils";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faXmarkLarge } from "@fortawesome/pro-solid-svg-icons";
+import * as RadixToast from "@radix-ui/react-toast";
+
+type _ToastProps = {
+  id: string;
+  content: ReactNode;
+  icon?: JSX.Element;
+  variant?: "info" | "danger" | "warning" | "success";
+  timer?: number;
+};
+export type ToastProps = _ToastProps &
+  Omit<React.HTMLProps<HTMLDivElement>, keyof _ToastProps>;
+
+export function Toast(props: ToastProps) {
+  const { content, icon, variant = "info", timer = 10000 } = props;
+  const timerCSS = {
+    "--bar-timer": `${timer / 1000}s`,
+  } as CSSProperties;
+
+  return (
+    <RadixToast.Root asChild duration={timer}>
+      <div className={classnames(styles.root)}>
+        <div className={classnames(styles.contentWrapper, getStyle(variant))}>
+          <RadixToast.Description asChild>
+            <div className={styles.content}>
+              <Icon wrapperClasses={styles.icon}>{icon}</Icon>
+              <div className={styles.message}>{content}</div>
+            </div>
+          </RadixToast.Description>
+          <div className={styles.toastProgress}>
+            <div className={styles.toastProgressBar} style={timerCSS} />
+          </div>
+          <RadixToast.Close className={styles.closeIconWrapper}>
```

Comment:
How about [them aria-labels](https://www.radix-ui.com/primitives/docs/components/toast#close-icon-button)?

Reply:
Ah, yep. Added it

Reply:
Just pointing out that the docs also use `aria-hidden` for the child of `Toast.Close`;

Reply:
As I'm not sure how `aria-hidden` is best applied here or with other similar places (places that use `Icon`), I'll add the figuring of aria labels as a separate task to the Cyberstorm Website Release folders lists.

Reply:
TS-1964